SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialMultiWorkerSettings.cpp

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/SpatialMultiWorkerSettings.h"
+
+#include "LoadBalancing/GridBasedLBStrategy.h"
+#include "LoadBalancing/LayeredLBStrategy.h"
+#include "LoadBalancing/OwnershipLockingPolicy.h"
+#include "Utils/LayerInfo.h"
+
+#include "Misc/MessageDialog.h"
+
+#define LOCTEXT_NAMESPACE "SpatialMultiWorkerSettings"
+
+#if WITH_EDITOR
+void UAbstractSpatialMultiWorkerSettings::PostEditChangeProperty(struct FPropertyChangedEvent& PropertyChangedEvent)
+{
+	Super::PostEditChangeProperty(PropertyChangedEvent);
+
+	const FName Name = (PropertyChangedEvent.MemberProperty != nullptr) ? PropertyChangedEvent.MemberProperty->GetFName() : NAME_None;
+
+	if (Name == GET_MEMBER_NAME_CHECKED(UAbstractSpatialMultiWorkerSettings, WorkerLayers))
+	{
+		ValidateNonEmptyWorkerLayers();
```

Comment:
Should all these validation functions be editor only?

Reply:
that makes sense, I didn't know that was a thing 😄

Reply:
Do you mean their declaration should be `WITH_EDITOR`? (the method implementation here is `WITH_EDITOR` already)